services/app/src/components/chessBoard/Square.jsx

```diff
@@ -13,14 +13,26 @@ const Square = ({
   filled
 }) => {
 
-  const { board } = useContext(ChessBoardContext);
+  const { board, findUserByUsername } = useContext(ChessBoardContext);
   const [showPopover, setShowPopover] = useState(false);
   const filteredFigures = items.filter((item, index, self) => index === self.findIndex((val) => val.img === item.img));
 
   const renderBubble = useCallback((item, key) => {
+    
+    const playerAvatarColor = (username) => {
+      const player = findUserByUsername(username);
+      return {
+        color: `rgb(${player.color.text.r}, ${player.color.text.g}, ${player.color.text.b})`,
```

Comment:
Create a new function `rgbToColor` which accepts object like this as a param
```
{
  r: <value>;
  g: <value>;
  b: <value>;
}
``` 

and returns rgb string `rgb(${r}, ${g}, ${b})`

Then it could be reused like this
```
  color: rgbToColor(player.color.text);
  backgroundColor: rgbToColor(player.color.background);
```

`rgbToColor` should be moved into some util folder and reused across the codebase.